optical/converter/converter.py

```diff
@@ -336,11 +336,6 @@ def convert_pascal(
     for split in splits:
         output_subdir = output_dir / split
         output_subdir.mkdir(parents=True, exist_ok=True)
-
-    images = df["image_id"].unique()
-    for image in images:
-        image_df = df.loc[lambda s: s["image_id"] == image]
```

Comment:
@bharatkumarreddy the syntax looks pretty complex. is there a special reason to use `lambda` function here instead of just `df.loc[df["image_id"] == image]` or `df.query("image_id == @image")`?